qemu/tests/ovs_enslave_port.py

```diff
@@ -0,0 +1,66 @@
+from avocado.utils import process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Enslave a port which is already a memeber of ovs into another
+
+    1) Check current ovs bridge and list the ports in it
+    2) Create a new ovs bridge
+    3) Enslave one port which is a member of current ovs bridge into the new one
+    4) Delete the new ovs bridge
+
+    :param test: Kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_ovs_ports(name):
+        """
+        Get ovs ports
+
+        :param name: ovs bridge name
+        """
+        cmd = "ovs-vsctl list-ports %s" % name
+        return process.system_output(cmd, shell=True).decode().splitlines()
+
+    def create_ovs_bridge(name):
+        """
+        Create ovs bridge
+
+        :param name: ovs bridge name
+        """
+        if name in process.system_output("ovs-vsctl list-br").decode():
+            cmd = "ovs-vsctl del-br %s" % name
+            process.system_output(cmd, shell=True)
+        cmd = "ovs-vsctl add-br %s" % name
+        process.system_output(cmd, shell=True)
+
+    def enslave_port(bridge, port):
+        """
+        Enslave port to bridge
+
+        :param bridge: ovs bridge name
+        :param port: port to enslave to the bridge
+        """
+        cmd = "ovs-vsctl add-port %s %s" % (bridge, port)
+        return process.system_output(cmd, shell=True).decode()
+
+    netdst = params.get("netdst")
+    if netdst not in process.system_output("ovs-vsctl list-br").decode():
+        test.error("%s isn't an openvswith bridge" % netdst)
```

Comment:
@xiywang How about using test.cancel instead? As "error" may indicate that there is an automation bug. However this is just a testing environment backend issue.